gems/pending/appliance_console/utilities.rb

```diff
@@ -49,8 +49,7 @@ def self.db_host_type_database
     end
 
     def self.pg_status
-      system("service #{PostgresAdmin.service_name} status > /dev/null 2>&1")
-      $?.exitstatus == 0 ? "running" : "not running"
+      AwesomeSpawn.run("pg_isready", :params => ["-q"]).success? ? "running" : "not running"
```

Comment:
This is good. It at least needed to change to `LinuxAdmin::Service.new(PostgresAdmin.service_name).status`, but if this is more portable, I'm fine with it.